examples/next/docs/vue3/demo/src/components/DataGrid.vue

```diff
@@ -0,0 +1,101 @@
+<script src="./DataGrid.js"></script>
```

Comment:
I think we could move the entire logic into this (`DataGrid.vue`) file to keep it consistent with the vue2 demo.